tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/reindexing/NodesSecondaryChildrenRelatedTests.java

```diff
@@ -0,0 +1,285 @@
+package org.alfresco.elasticsearch.reindexing;
+
+import static org.alfresco.utility.report.log.Step.STEP;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.alfresco.elasticsearch.SearchQueryService;
+import org.alfresco.rest.core.RestWrapper;
+import org.alfresco.rest.model.RestNodeBodyMoveCopyModel;
+import org.alfresco.rest.model.RestNodeModel;
+import org.alfresco.tas.AlfrescoStackInitializer;
+import org.alfresco.utility.data.DataContent;
+import org.alfresco.utility.data.DataSite;
+import org.alfresco.utility.data.DataUser;
+import org.alfresco.utility.model.ContentModel;
+import org.alfresco.utility.model.FileModel;
+import org.alfresco.utility.model.FileType;
+import org.alfresco.utility.model.FolderModel;
+import org.alfresco.utility.model.SiteModel;
+import org.alfresco.utility.model.UserModel;
+import org.alfresco.utility.network.ServerHealth;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+@ContextConfiguration(locations = "classpath:alfresco-elasticsearch-context.xml",
+    initializers = AlfrescoStackInitializer.class)
+public abstract class NodesSecondaryChildrenRelatedTests extends AbstractTestNGSpringContextTests
+{
+    protected static final String A = "A", B = "B", C = "C", K = "K", L = "L", M = "M", P = "P", Q = "Q", R = "R", S = "S", X = "X", Y = "Y", Z = "Z";
```

Comment:
I think this is a reasonable use of multiple declarations on a single line even if PMD doesn't. I suggest marking it with `@SuppressWarnings("PMD.whatever")`.

Reply:
Good hint - the annotation - thanks!